packages/lib/validateBookerLayouts.ts

```diff
@@ -0,0 +1,29 @@
+import { bookerLayoutOptions, type BookerLayoutSettings } from "@calcom/prisma/zod-utils";
+
+export const validateBookerLayouts = (settings: BookerLayoutSettings) => {
```

Comment:
Reusable validation function that's used when submitting the forms.

Reply:
I mentioned this where it was used too - is there any reason we dont do this on the backend?

Reply:
We do it both actually. I think the faster we can give errors, the better?  That's at least why it's a shared utility, so we can easily run it in both places.